### PR TITLE
Cucumber tests

### PR DESCRIPTION
**Pull request by zjb0807**

### CODE REVIEW

**Comment by xlc** [comment]
can you do a rebase master?

**Comment by zjb0807** [comment]
> can you do a rebase master?

After https://github.com/laminar-protocol/laminar-chain/pull/145 merged. I modified on the PR.

**Comment by zjb0807** [comment]
https://github.com/laminar-protocol/laminar-chain/compare/master...cucumber
Use the link to review changes.

File: runtime/src/tests.rs [hunk 1]
```diff
@@ -148,19 +142,119 @@ pub fn one_percent() -> Fixed128 {
...
+pub fn negative_one_percent() -> Fixed128 {
+	Fixed128::recip(&Fixed128::from_natural(-100)).unwrap()
 }
 
 pub fn multi_currency_balance(who: &AccountId, currency_id: CurrencyId) -> Balance {
 	<Runtime as synthetic_protocol::Trait>::MultiCurrency::free_balance(currency_id, &who)
 }
 
+pub fn synthetic_create_pool() -> DispatchResult {
+	SyntheticLiquidityPools::create_pool(origin_of(&POOL::get()))?;
```
Comment (xlc) [inline runtime/src/tests.rs:154]: why create twice?

Comment (zjb0807) [reply runtime/src/tests.rs:154]: Do test cases that have multiple pools. I'll remove it first.

**Pull request merged**